src/subsidy.cpp

```diff
@@ -107,6 +111,26 @@ std::pair<NewsReferenceType, NewsReferenceType> SetupSubsidyDecodeParam(const Su
 	}
 	SetDParam(5, s->dst);
 
+	/* If the subsidy is being offered or awarded, add the right string to communicate the duration. */
+	if (mode == SUBSIDY_NEWS_OFFERED) {
+		if (_settings_game.difficulty.subsidy_duration == 1) {
+			SetDParam(7, STR_NEWS_SERVICE_SUBSIDY_OFFERED_ONE_YEAR);
+		}
+		else {
```

Comment:
```suggestion
		} else {
```
codestyle (also a few other places)